pom.xml

```diff
@@ -6,7 +6,7 @@
 
     <groupId>io.xpring</groupId>
     <artifactId>xpring4j</artifactId>
-    <version>6.1.0</version>
+    <version>HEAD-SNAPSHOT</version>
```

Comment:
Best practice is to leave `master` as a SNAPSHOT so that CI can automatically deploy periodic (e.g., nightly) SNAPSHOTS to sonatype.